tests/BinTest.php

```diff
@@ -86,4 +86,13 @@ public function test17NoSkipOfReloadCommand()
         $this->assertFileExists('ShouldNotExist');
         unlink('ShouldNotExist');
     }
+
+    public function test19ErrorIfCommandExecutionFailed()
+    {
+        chdir(__DIR__ . '/fixtures/19-error-if-command-execution-failed');
```

Comment:
Looks like this is a a duplicate of fixture 18? Maybe just use this one here?

Reply:
Is obsolete....